docs/tutorial/arguments/optional.md

```diff
@@ -113,7 +113,7 @@ name: str
 
 Now, finally what we came for, an optional *CLI argument*.
 
-To make a *CLI argument* optional, use `typer.Argument()` and pass a different "default" as the first parameter to `typer.Argument()`, for example `None`:
+To make a *CLI argument* optional, use `typer.Argument()` and make sure to provide a "default" value, for example `None`:
```

Comment:
I've now rewritten it so that it covers both `Annotated` and non-Annotated code snippets:
```
name: Optional[str] = typer.Argument(default=None)
```
&
```
name: Annotated[Optional[str], typer.Argument()] = None
```

Reply:
From the point of someone, who were reading the tutorial for the first time (me), that's still confusing if you go from the previous examples:
> ```Python
> name: Annotated[str, typer.Argument()]
> ```
> ...that works exactly the same as
> ```Python
> name: str
> ```

where you only see `typer.Argument()` used inside the type definition. And peeking in the next example from the *phrase* in subject, you see only:

> Now we have:
> ```Python
> name: Annotated[Optional[str], typer.Argument()] = None
> ```


So I'd suggest adding both variants you mentioned in tutorial, like:

> ```Python
> name: Annotated[Optional[str], typer.Argument()] = None
> ```
> ...that works exactly the same as
> ```Python
> name: Optional[str] = typer.Argument(default=None)
> ```

And placing the *Annotated* version first as it is recommended in the rest of the tutorial.

Reply:
Hm, I see.

I wonder whether it would be less confusing if the first example of an optional argument doesn't actually use `None` as default, but some other random value. I think that would help enforce the idea that the default is all that matters, and that `Optional` is only used to support `None` as a potential value. What do you think?